### PR TITLE
making SecorConfig method public & use timezone from config for dateparser

### PR DESCRIPTION
**Pull request by lovenishgoyal**
Hi,

While I was trying to build project one of jUnit test case for Iso8601ParserTest.java class was failing. My timezone is GMT+05:30 and when it was processing one "timestamp":"2016-03-02T18:36:14+00:00" it is converting this into "2016-03-03T00:06:14+00:00"(As per GMT). I have modified the code based on configure timezone.

Please check and let me know in case of more clarification.

Thanks,
Lovenish

### CODE REVIEW

**Comment by lovenishgoyal** [comment]
Hi @HenryCaiHaiying , I am able to build the project with these new changes in my local, Could you please guide me about next step.
Thanks in advance

File: src/main/java/com/pinterest/secor/parser/DateMessageParser.java [hunk 1]
```diff
@@ -41,9 +42,17 @@
     private static final Logger LOG = LoggerFactory.getLogger(DateMessageParser.class);
     protected static final String defaultDate = "dt=1970-01-01";
     protected static final String defaultFormatter = "yyyy-MM-dd";
-
+    SimpleDateFormat outputFormatter = new SimpleDateFormat(defaultFormatter);
```
Comment (HenryCaiHaiying) [inline src/main/java/com/pinterest/secor/parser/DateMessageParser.java]: These variables should be either private or protected.

File: src/main/java/com/pinterest/secor/common/SecorConfig.java [hunk 2]
```diff
@@ -416,32 +416,32 @@ public boolean getBoolean(String name) {
...
-    private void checkProperty(String name) {
+    public void checkProperty(String name) {
         if (!mProperties.containsKey(name)) {
             throw new RuntimeException("Failed to find required configuration option '" +
                                        name + "'.");
         }
     }
 
-    private String getString(String name) {
+    public String getString(String name) {
```
Comment (HenryCaiHaiying) [inline src/main/java/com/pinterest/secor/common/SecorConfig.java]: Do you need to change these accessor methods to public?

**Comment by HenryCaiHaiying** [comment]
Have you looked at the test failure in CI?

File: src/main/java/com/pinterest/secor/parser/DateMessageParser.java (hunk 1 above)
Comment (lovenishgoyal) [inline src/main/java/com/pinterest/secor/parser/DateMessageParser.java]: I'll make the change.

File: src/main/java/com/pinterest/secor/common/SecorConfig.java (hunk 2 above)
Comment (lovenishgoyal) [inline src/main/java/com/pinterest/secor/common/SecorConfig.java]: these are common methods so we can use from our customized classes

**Comment by lovenishgoyal** [comment]
CI is not showing any error, Build got failed while it was downloading jars. Test cases did not even start.
https://travis-ci.org/pinterest/secor/jobs/117530772

Can we start the build once again ?

**Comment by lovenishgoyal** [comment]
@HenryCaiHaiying - Ping

**Comment by HenryCaiHaiying** [comment]
Restarting the CI build

**Comment by lovenishgoyal** [comment]
Hi @HenryCaiHaiying, I have changed Access Modifier for variables.

**Comment by HenryCaiHaiying** [comment]
Looks good, thanks.

**Pull request merged**